package/contents/ui/main.qml

```diff
@@ -221,7 +248,12 @@ Item {
 
 				// console.log('[commandoutput]', 'stdout', JSON.stringify(stdout))
 				// console.log('[commandoutput]', 'format', JSON.stringify(formattedText))
-				widget.outputText = formattedText
+
+				if (cmd == config.command) {
+					root.outputText = formattedText
+				} else if (cmd == config.tooltipCommand) {
+					root.tooltipText = formattedText
+				}
```

Comment:
Hmm, the code above this probably needs to be put into a function if it's "reused" so that this block of code focuses on the if/else logic of the outputText/tooltipText.